bugsnag/tornado/__init__.py

```diff
@@ -42,7 +42,7 @@ def add_tornado_request_to_notification(self, event: bugsnag.Event):
         event.add_tab("request", request_tab)
 
         if bugsnag.configure().send_environment:
-            env = WSGIContainer.environ(self.request)
+            env = WSGIContainer(self.application).environ(self.request)
```

Comment:
Can you back out this change from this PR? I'm happy to merge these PRs with CI failing since we know the cause and can fix it separately before releasing a new version